docs/updating/8-0.md

```diff
@@ -0,0 +1,182 @@
+:::
+
+:::info Breaking Changes
+For a **complete list of breaking changes** from Ionic 7 to Ionic 8, please refer to [the breaking changes document](https://github.com/ionic-team/ionic-framework/blob/feature-8.0/BREAKING.md#version-8x) in the Ionic Framework repository.
+:::
+
+## Getting Started
+
+### Angular
+
+1. Ionic 8 supports Angular 16+. Update to the latest version of Angular by following the [Angular Update Guide](https://update.angular.io/).
+
+2. Update to the latest version of Ionic 8:
+
+```shell
+npm install @ionic/angular@next
+```
+
+If you are using Ionic Angular Server and Ionic Angular Toolkit, be sure to update those as well:
+
+```shell
+npm install @ionic/angular@next @ionic/angular-server@next @ionic/angular-toolkit@11
+```
+
+> Note: `@ionic/angular-toolkit@11` requires a minimum of Angular 17. If you are still on Angular 16, then you may want to only update to to `@ionic/angular-toolkit@10` instead.
+
+3. Update any `IonBackButtonDelegate` imports from `@ionic/angular` to import `IonBackButton` from `@ionic/angular` instead.
+
+### React
+
+1. Ionic 8 supports React 17+. Update to the latest version of React:
+
+```shell
+npm install react@latest react-dom@latest
+```
+
+2. Update to the latest version of Ionic 8:
+
+```shell
+npm install @ionic/react@next @ionic/react-router@next
+```
+
+### Vue
+
+1. Ionic 7 supports Vue 3.0.6+. Update to the latest version of Vue:
+
+```shell
+npm install vue@latest vue-router@latest
+```
+
+3. Update to the latest version of Ionic 8:
+
+```shell
+npm install @ionic/vue@next @ionic/vue-router@next
+```
+
+### Core
+
+1. Update to the latest version of Ionic 8:
+
+```shell
+npm install @ionic/core@next
+```
+
+## Recommended Changes
+
+The following changes are not required to update to Ionic 8 as your application will continue to work. However, we recommend making the following changes to ensure you can use the new features in Ionic 8.
+
+### Dark Theme
+
+In previous versions, it was recommended to define the dark theme in the following way:
+
+```css
+@media (prefers-color-scheme: dark) {
+  body {
+    /* global app variables */
+  }
+
+  .ios body {
+    /* global ios app variables */
+  }
+
+  .md body {
+    /* global md app variables */
+  }
+}
+```
+
+In Ionic Framework version 8, the dark theme is being distributed via css files that can be imported. Below is an example of importing a dark theme file in Angular:
+
+```css
+/* @import '@ionic/angular/css/themes/dark.always.css'; */
+/* @import "@ionic/angular/css/themes/dark.class.css"; */
+@import "@ionic/angular/css/themes/dark.system.css";
+```
+
+Notice that the dark theme is now applied to the `:root` selector instead of the `body` selector. The [`:root`](https://developer.mozilla.org/en-US/docs/Web/CSS/:root) selector represents the `<html>` element and is identical to the selector `html`, except that its specificity is higher.
```

Comment:
```suggestion
The dark theme is now applied to the `:root` selector instead of the `body` selector. The [`:root`](https://developer.mozilla.org/en-US/docs/Web/CSS/:root) selector represents the `<html>` element and is identical to the selector `html`, except that its specificity is higher.
```

The "notice that" part doesn't make sense when there isn't any CSS showing the `:root` selector